docs/csharp/methods.md

```diff
@@ -34,7 +34,7 @@ This topic contains the following sections:
 - [Expression-bodied members](#expr)
 - [Iterators](#iterators)
 
-<a name="signatures" /a>
+<a name="signatures" />
```

Comment:
> Who knows what kind of crazy browsers out there might try to parse this and choke.

I think browsers are not relevant here, since the generated HTML does not actually have self-closing tags.

For example, `<a name="invocation" />` later in this file is rendered as: `<p><a name="invocation"></a></p>`.